code/modules/modular_computers/computers/item/computer.dm

```diff
@@ -31,6 +31,7 @@
 	var/steel_sheet_cost = 5								// Amount of steel sheets refunded when disassembling an empty frame of this computer.
 
 	obj_integrity = 100
+	integrity_failure = 50
```

Comment:
@phil235 
https://github.com/tgstation/tgstation/blob/master/code/modules/modular_computers/computers/item/computer.dm#L181
and
https://github.com/tgstation/tgstation/blob/master/code/modules/modular_computers/computers/item/computer.dm#L195

It does use this value.